src/x11_clipboard.rs

```diff
@@ -533,12 +460,8 @@ impl Manager {
 			}
 
 			if manager!().window != 0 {
-				unsafe {
-					xcb::ffi::xproto::xcb_destroy_window(
-						shared!().conn.as_ref().unwrap().get_raw_conn(),
-						manager!().window,
-					)
-				};
+				let window = manager!().window;
+				shared!().conn.as_ref().unwrap().destroy_window(window);
 				shared!().conn.as_ref().unwrap().flush();
```

Comment:
These should be ignored. (Definitely *not* unwrapped because this is called from a `drop`, and we probably don't want to panic while potentially another panic is happening)
```suggestion
				let _ = shared!().conn.as_ref().unwrap().destroy_window(window);
				let _ = shared!().conn.as_ref().unwrap().flush();
```

Reply:
Done.